TestApp/ios/Podfile.lock

```diff
@@ -294,8 +294,9 @@ DEPENDENCIES:
   - Yoga (from `../node_modules/react-native/ReactCommon/yoga`)
 
 SPEC REPOS:
-  trunk:
+  https://cdn.cocoapods.org/:
     - AppCenter
+  https://github.com/CocoaPods/Specs.git:
```

Comment:
@AnastasiaKubova  After running **./update-npm-packages.sh**  , this will change automatically. After removing this change, TestApp-iOS cannot run locally. Please check if changes are needed here?

Reply:
Let's revert these lines (297 & 299) to the `trunk` repo as it a modern default CDN alias.
https://github.com/microsoft/appcenter-sdk-react-native/pull/814#discussion_r395354750

Reply:
Reverted them.